ygot/struct_validation_map_test.go

```diff
@@ -1632,6 +1683,14 @@ type validatedMergeTestSliceField struct {
 	String *string
 }
 
+type validatedMergeTestWithStructField struct {
+	StructField reflect.StructField
```

Comment:
This is a bit strange as a test case -- `reflect.StructField` is usually formed by doing `reflect.ValueOf` something -- `ygen` can never produce this form of struct. Could you explain which of the features that you were trying to cover here please?

Reply:
In your earlier comment, you had asked me add testcases for StructField.

Reply:
Sorry -- I meant something different - i.e., a field of a struct, not a `reflect.StructField`. Essentially, I was asking whether we have a test whereby we have:

```
src := ExampleStruct{S: "hello"}
dst := ExampleStruct{S: "world"}
```

and we check that the result is that the `S` string field is overwritten.

Apologies for the misunderstanding.